lhotse/supervision.py

```diff
@@ -1,14 +1,68 @@
 import logging
 from dataclasses import dataclass
 from itertools import islice
-from typing import Any, Callable, Dict, Iterable, List, Mapping, Optional, Sequence
+from typing import Any, Callable, Dict, Iterable, List, Mapping, NamedTuple, Optional, Sequence
 
 from lhotse.serialization import Serializable
 from lhotse.utils import Seconds, asdict_nonull, exactly_one_not_null, fastcopy, \
     ifnone, index_by_id_and_check, \
     perturb_num_samples, split_sequence
 
 
+@dataclass(frozen=True, unsafe_hash=True)
+class AlignmentItem(NamedTuple):
+    symbol: str
+    start: Seconds
+    duration: Seconds
+
+    @property
+    def end(self) -> Seconds:
+        return round(self.start + self.duration, ndigits=8)
+
+    def with_offset(self, offset: Seconds) -> 'AlignmentItem':
+        """Return an identical ``AlignmentItem``, but with the ``offset`` added to the ``start`` field."""
+        return fastcopy(self, start=round(self.start + offset, ndigits=8))
```

Comment:
I think `fastcopy` is not going to work on a named tuple, since it has only 3 members I think it's better to explicitly instantiate a new named tuple (i.e. `AlignmentItem(self[0], round(...), self[2])` - IIRC indexing is faster than named attr access and this code is potentially executed many times on large supervision/cut sets)